readme.md

```diff
@@ -11,6 +11,14 @@ Development is focused on the next major version in the `master` branch. No more
 -   The [`master`](https://github.com/SFML/SFML/tree/master) branch contains work in progress for the next major version SFML 3. As such it's considered unstable, but any testing and feedback is highly appreciated.
 -   The [`2.6.0`](https://github.com/SFML/SFML/tree/2.6.0) tag is the latest official SFML release and will be the last minor release in the 2.x series.
 
+## CMake Template
+
+https://github.com/SFML/cmake-sfml-project
```

Comment:
I'd put the link below the paragraph

Reply:
The rest of the README uses hyperlinks so I chose to instead use that instead of having the raw URL visible as text.